### PR TITLE
Some minor edits.

### PR DESCRIPTION
**Pull request by christianpaquin**
Some minor edits:
- consistency of hyphenation in base64url-encode(d)
- use transaction instead of use case in one place because selective might be needed or not within one use case, depending on the interaction with RP
- explicitly called out period character to avoid misinterpreting as missing word
- some typos

### CODE REVIEW

**Review by Sakurann — APPROVED** [review]

File: main.md [hunk 1]
```diff
@@ -595,7 +595,7 @@ Verifiers MUST go through (at least) the following steps before
 trusting/using any of the contents of an SD-JWT:
 
  1. Determine if holder binding is to be checked for the SD-JWT. Refer to (#holder_binding_security) for details.
- 2. Check that the presentation consists of six `.`-separated elements; if holder binding is not required, the last element can be empty.
+ 2. Check that the presentation consists of six period-separated (`.`) elements; if holder binding is not required, the last element can be empty.
```
Comment (Sakurann) [inline main.md:598]:
```suggestion
 2. Check that the presentation consists of four period-separated (`.`) elements; if holder binding is not required, the last element can be empty.
```

**Comment by Sakurann** [comment]
changed six . separated to four per Christian's separate comment

**Comment by danielfett** [comment]
> changed six . separated to four per Christian's separate comment

What did I miss? The presentation should be six dot separated elements. Three for the SD-JWT, three for the SD-JWT-R.

**Comment by christianpaquin** [comment]
> What did I miss? The presentation should be six dot separated elements. Three for the SD-JWT, three for the SD-JWT-R.

Why would you need to merge both? I don't understand the proposed flow between issuer, holder, and verifier; perhaps a data flow section would help. I would think the holder gets a 3+1 part JWS with SVC (which can be `.`-concatenated). Then all you need to present to a verifier is the same data, along with some user proof key signature if the JWT is bound to a user key (for which a property is added in the SD-JWT-R, if I understood correctly; more details would be helpful here).

File: main.md [hunk 2]
```diff
@@ -595,7 +595,7 @@ Verifiers MUST go through (at least) the following steps before
 trusting/using any of the contents of an SD-JWT:
 
  1. Determine if holder binding is to be checked for the SD-JWT. Refer to (#holder_binding_security) for details.
- 2. Check that the presentation consists of six `.`-separated elements; if holder binding is not required, the last element can be empty.
+ 2. Check that the presentation consists of four period-separated (`.`) elements; if holder binding is not required, the last element can be empty.
```
Comment (Sakurann) [inline main.md:598]:
```suggestion
 2. Check that the presentation consists of six period-separated (`.`) elements; if holder binding is not required, the last element can be empty.
```

**Comment by Sakurann** [comment]
creating a separate issue on six vs four period separated elements to merge this editorial PR

**Pull request merged**